haystack/components/embedders/sentence_transformers_document_embedder.py

```diff
@@ -61,6 +61,7 @@ def __init__(  # noqa: PLR0913 # pylint: disable=too-many-positional-arguments
         precision: Literal["float32", "int8", "uint8", "binary", "ubinary"] = "float32",
         encode_kwargs: Optional[dict[str, Any]] = None,
         backend: Literal["torch", "onnx", "openvino"] = "torch",
+        revision: Optional[str] = None,
```

Comment:
This should fix the issue with the linter
```suggestion
        backend: Literal["torch", "onnx", "openvino"] = "torch",
        *,
        revision: Optional[str] = None,
```